src/_cffi_src/openssl/pem.py

```diff
@@ -82,7 +82,11 @@
                                unsigned char *, int, pem_password_cb *,
                                void *);
 int PEM_write_bio_DHparams(BIO *, DH *);
+int PEM_write_bio_DHxparams(BIO *, DH *);
 """
 
 CUSTOMIZATIONS = """
+#ifndef EVP_PKEY_DHX
```

Comment:
I added a guard but not exactly the one you suggested. The `!` before the `defined(EVP_PKEY_DHX)` should not be there.